input/metadata/properties/default.json

```diff
@@ -3871,6 +3871,18 @@
           "type": "number",
           "description": "Cost of Feed (US dollars/kg DM)",
           "minimum": 0
+        },
+        "shrink_factor": {
+          "type": "number",
+          "description": "Shrinkage factor for purchased feed.",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "buffer": {
+          "type": "number",
+          "description": "Buffer factor for purchased feed.",
+          "minimum": 0,
+          "maximum": 1
```

Comment:
I think I'd add defaults here of the 10% shrink and 15% buffer so it's not required that a user have a specific buffer and shrink for each individual feed if they don't want.